tests/specs/graph/JsrSpecifiers_FastCheck_Enums.txt

```diff
@@ -108,3 +108,28 @@ Fast check https://jsr.io/@scope/a/1.0.0/mod.ts:
   enum NonExportedEnum {
     Value
   }
+  --- DTS ---
+  export declare enum EnumWithNoInits {
+    Value1,
+    Value2
+  }
+  export declare enum EnumWithNumInits {
+    Value1 = 1,
+    Value2 = 2
+  }
+  export declare enum EnumWithStringInits {
+    Value1 = "a",
+    Value2 = "b"
+  }
+  declare const value: number;
+  export declare enum EnumWithNonConstInits {
+    Value1 = new Public1().test,
+    Value2 = new Public2().asdf * value + NonExportedEnum.Value
```

Comment:
Some expressions here that we'll need to handle somehow (seems like we can just remove them)